src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -453,6 +453,12 @@ Status GcsActorManager::CreateActor(const ray::rpc::CreateActorRequest &request,
     return Status::Invalid("Actor may be already destroyed.");
   }
 
+  const auto &actor_name = iter->second->GetName();
```

Comment:
The invariant here is that this RPC path is not called if the actor is not registered due to conflicting names.